src/tests/ftest/nvme/pool_exclude.py

```diff
@@ -99,13 +183,29 @@ def run_nvme_pool_exclude(self, num_pool, oclass=None):
                 self.log.info("Pool Version after exclude %s", pver_exclude)
                 # Check pool version incremented after pool exclude
                 self.assertTrue(pver_exclude > pver_begin, "Pool Version Error:  After exclude")
+
                 # Wait to finish the threads
                 for thrd in threads:
                     thrd.join()
-                    if not self.out_queue.empty():
-                        self.assert_on_exception()
+                errors = 0
+                while not thread_queue.empty():
+                    result = thread_queue.get()
+                    self.log.debug("Results from thread %s (log %s)", result["job_id"],
+                                   result["log"])
+                    self.log.debug(result["result"])
```

Comment:
Since the `CmdResult.stdout` isn't very readable with this approach:
```
2023-02-16 17:13:18,090 pool_exclude     L0194 DEBUG| Results from thread 0 (log ior_thread_write_pool_0_test_0.log)
2023-02-16 17:13:18,090 pool_exclude     L0195 DEBUG| command: '/usr/lib64/mpich/bin/mpirun -genv COVFILE=/tmp/test.cov -genv D_LOG_FILE=/var/tmp/daos_testing/ior_thread_write_pool_0_test_0.log -genv MPI_LIB="" -genv FI_PSM2_DISCONNECT=1 -hostfile /var/tmp/avocado_aba2a2be/avocado_job_hvhclavk/1-._nvme_pool_exclude.py_NvmePoolExclude.test_nvme_pool_excluded_run-aggregation-checksum-container-hosts-ior-client_processes-iorflags-loop_test-pool-rebuild-server_config-engines-0-storage-0-1-setup-test_obj_class-8432/hostfile51384 -np 48 ior -a DFS -b 50000000 -w -F -k -G 1 -i 10 -o /testFile -t 4000 --dfs.chunk_size 1048576 --dfs.cont C60A4C00-0C41-42F1-A6C5-4350EB1DCEAB --dfs.dir_oclass RP_3G6 --dfs.group daos_server --dfs.oclass RP_3G6 --dfs.pool 628295D9-26E2-4EFA-AC4A-95A1523609A5'
exit_status: 0
duration: 29.476590633392334
interrupted: False
pid: 42758
encoding: 'UTF-8'
stdout: b'IOR-3.4.0+dev: MPI Coordinated Test of Parallel I/O\nBegan               : Thu Feb 16 17:11:40 2023\nCommand line        : ior -a DFS -b 50000000 -w -F -k -G 1 -i 10 -o /testFile -t 4000 --dfs.chunk_size 1048576 --dfs.cont C60A4C00-0C41-42F1-A6C5-4350EB1DCEAB --dfs.dir_oclass RP_3G6 --dfs.group daos_server --dfs.oclass RP_3G6 --dfs.pool 628295D9-26E2-4EFA-AC4A-95A1523609A5\nMachine             : Linux wolf-115.wolf.hpdd.intel.com\nTestID              : 0\nStartTime           : Thu Feb 16 17:11:41 2023\nPath                : /testFile.00000000\nFS                  : 3.2 TiB   Used FS: 5.1%   Inodes: -0.0 Mi   Used Inodes: 0.0%\n\nOptions: \napi                 : DFS\napiVersion          : DAOS\ntest filename       : /testFile\naccess              : file-per-process\ntype                : independent\nsegments            : 1\nordering in a file  : sequential\nordering inter file : no tasks offsets\nnodes               : 3\ntasks               : 48\nclients per node    : 16\nrepetitions         : 10\nxfersize            : 4000 bytes\nblocksize           : 47.68 MiB\naggregate filesize  : 2.24 GiB\n\nResults: \n\naccess    bw(MiB/s)  IOPS       Latency(s)  block(KiB) xfer(KiB)  open(s)    wr/rd(s)   close(s)   total(s)   iter\n------    ---------  ----       ----------  ---------- ---------  --------   --------   --------   --------   ----\nwrite     631.44     166534     0.000189    48828      3.91       0.039941   3.60       1.49       3.62       0   \nwrite     888.23     232860     0.000204    48828      3.91       0.001039   2.58       0.534962   2.58       1   \nwrite     890.87     233557     0.000187    48828      3.91       0.000924   2.57       0.538306   2.57       2   \nwrite     890.20     233383     0.000206    48828      3.91       0.000899   2.57       0.564306   2.57       3   \nwrite     847.15     222093     0.000209    48828      3.91       0.003321   2.70       0.618843   2.70       4   \nwrite     846.09     221819     0.000210    48828      3.91       0.000771   2.70       0.863014   2.71       5   \nwrite     885.48     232149     0.000176    48828      3.91       0.000837   2.58       0.481632   2.58       6   \nwrite     858.80     225152     0.000201    48828      3.91       0.000842   2.66       0.614708   2.67       7   \nwrite     823.46     215882     0.000213    48828      3.91       0.000941   2.78       0.759198   2.78       8   \nwrite     849.37     222674     0.000216    48828      3.91       0.000838   2.69       0.663775   2.69       9   \n\nSummary of all tests:\nOperation   Max(MiB)   Min(MiB)  Mean(MiB)     StdDev   Max(OPs)   Min(OPs)  Mean(OPs)     StdDev    Mean(s) Stonewall(s) Stonewall(MiB) Test# #Tasks tPN reps fPP reord reordoff reordrand seed segcnt   blksiz    xsize aggs(MiB)   API RefNum\nwrite         890.87     631.44     841.11      73.34  233537.27  165528.34  220491.82   19226.26    2.74731         NA            NA     0     48  16   10   1     0        1         0    0      1 50000000     4000    2288.8 DFS      0\nFinished            : Thu Feb 16 17:12:09 2023\n'
stderr: ''
```
I suggest we do this instead:
```suggestion
                    for name in ("command", "exit_status", "interrupted", "duration"):
                        self.log.debug("  %s: %s", name, getattr(result["result"], name))
                    for name in ("stdout", "stderr"):
                        self.log.debug("  %s:", name)
                        for line in getattr(result["result"], name).splitlines()
                            self.log.debug("    %s:", line)
```